src/features/wallet/root/ActivityCard/ActivityCard.tsx

```diff
@@ -57,14 +56,9 @@ const ActivityCard = forwardRef((props: Props, ref: Ref<BottomSheet>) => {
   const {
     activity: { txns, filter },
   } = useSelector((state: RootState) => state)
+  const hasActivity = txns.all.data.length !== 0
   const sheet = useRef<BottomSheet>(null)
 
-  useEffect(() => {
```

Comment:
I had to move this to WalletView in order to determine if the account had any activity or not

Reply:
See https://github.com/helium/hotspot-app/pull/149/files#diff-356b4c10b385bf10218d349d63f2d7186b7c3ea69b220b18b4359289cf4bae25R58